.gitignore

```diff
@@ -28,4 +28,4 @@ helpstack.conf
 
 # backup files
 *~
-Accounts.json
+
```

Comment:
If Accounts.json is supposed to live in a specific part of the path, I can just put it there, and you won't have to do anything special for android.

Reply:
@leonardr 
That sounds okay to me.
It would be in `/simplified-app-simplye/src/main/assets`